### ISSUE TITLE
Get loklak account apps working

### ISSUE THREAD

**Issue by mariobehling** [fossasia/apps.loklak.org#109]
The loklak account apps have not been working yet. Please help to make them work.

**Comment by djmgit** [comment]
@mariobehling I would like to work on this

**Comment by Abhi2424shek** [comment]
Go ahead

**Comment by djmgit** [comment]
@mariobehling I was going through the signup app and I observed the following :

On hitting the api : http://api.loklak.org/api/signup.json?signup=abc@def.com&password=xyz
I am getting the following response : 403 Public signup disabled. I am getting the same error when I am calling the api on local server. I guess this is preventing the app from functioning. Is this error justified and thrown on purpose by the server?

**Comment by mariobehling** [comment]
@Orbiter Any idea?

**Comment by Orbiter** [comment]
The sign-up is actually disabled. I am currently unable to active it because someone has added a configuration save method which ignores UTF8 an that kills the config file.
I am working on it.

**Comment by djmgit** [comment]
@Orbiter is there any test user credential using which I can test the login app and other account apps?

**Comment by Orbiter** [comment]
The actual problem was lately that I had to migrate to the gradle build version and could not do without conflicts. Therefore I am now doing a fresh install on the production site and that required a ful back-up. The back-up was running for hours yesterday and today I hope that I am able to run the latest version including the UTF8 fix, so everything should work as expected.

**Comment by Orbiter** [comment]
it's running now with signup enabled!

**Comment by djmgit** [comment]
@Orbiter yeah its working on api.loklak.org but is the signup api only functional for api.loklak.org?
Because I tried calling the api on my local server (127.0.0.1:9000/api/signup.json) its showing public signup disabled.

**Comment by Orbiter** [comment]
oh on your own server you must configure the signup parameters yourself. They are not active by default.
you must set users.public.signup=true and to get confirmation mails upon signup, you have to set all the smtp.* parameters.

**Comment by djmgit** [comment]
@Orbiter okay :)

**Comment by djmgit** [comment]
@mariobehling @daminisatya  Presently on a successful login (via the login app) the window location is changed to /apps/applist/index.html but as of now there is no applist directory in the server html directory nither is there any index.html file in the applist directory present at apps.loklak.org root directory.
So after user makes a successful login where should I redirect him

**Comment by djmgit** [comment]
@Orbiter  The account apps are working now, I will be uploading a PR soon.However there are two issues while testing Password recovery app and reset password app. The password recovery app on sending request (clicking submit) is showing mail sending disabled and the reset password app requires a token parameter in the apps URL. How do i generate this token?
